src/consts/Stages.ts

```diff
@@ -1,44 +1,73 @@
-export const STAGES = [
+interface STAGE {
+  key: string;
+  text: string;
+  next: string;
+  nextEventTitle?: string;
+  previous: string;
+  previousEventTitle?: string;
```

Comment:
Should we make next and previous also be optional?  Will require updating other places in the code where it checks for empty string to check truthiness/presence -- but I think it make sense to have it "undefined" if there is none.